pkg/util/mount/mount.go

```diff
@@ -325,7 +325,7 @@ func pathWithinBase(fullPath, basePath string) bool {
 	if err != nil {
```

Comment:
This function is also implemented in Kubelet, although I'm not sure it's used anywhere? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/util/util.go#L51

Reply:
I don't see any uses of that method.

Reply:
removed the unused function. similar faulty logic exists in `pkg/util/mount/mount_windows.go` lockAndCheckSubPathWithoutSymlink and findExistingPrefix